.github/workflows/qa.yml

```diff
@@ -54,6 +61,13 @@ jobs:
           ignore-unfixed: true
           vuln-type: 'os'
           severity: 'CRITICAL,HIGH'
+      - name: Notify failures on Slack
```

Comment:
I don't know how munch sense this notification makes. this is a quality gate prior to merging something new. yes this will be run on the default branch as well but in the end the docker hub will build the final images and not this GHA. could you explain why it would make sense to you?